src/Compilers/Core/Portable/Compilation/Compilation.cs

```diff
@@ -1776,6 +1776,18 @@ private static EmitResult ToEmitResultAndFree(DiagnosticBag diagnostics, bool su
             bool emitPortablePdb = moduleBeingBuilt.EmitOptions.DebugInformationFormat == DebugInformationFormat.PortablePdb;
             string pdbPath = (pdbStreamProvider != null) ? (moduleBeingBuilt.EmitOptions.PdbFilePath ?? FileNameUtilities.ChangeExtension(SourceModule.Name, "pdb")) : null;
 
+            // The PDB path is emitted in it's entirety into the PE.  This makes it impossible to have deterministic
```

Comment:
Do you mean the fact that the path is passed to PdbWriter? PdbWriter always writes to a stream. It doesn't write to the specified path. It needs the path just because the implementation is odd.